apps/condo/pages/employee/[id]/index.tsx

```diff
@@ -282,39 +308,51 @@ export const EmployeeInfoPage: PageComponentType = () => {
     const { query } = useRouter()
     const { link } = useOrganization()
     const intl = useIntl()
-    const UpdateEmployeeMessage = intl.formatMessage({ id: 'employee.UpdateTitle' })
+    const LoadingInProgressMessage = intl.formatMessage({ id: 'LoadingInProgress' })
     const ErrorMessage = intl.formatMessage({ id: 'errors.LoadingError' })
+    const NotFoundMsg = intl.formatMessage({ id: 'NotFound' })
 
     const employeeId = String(get(query, 'id', ''))
+
     const { obj: employee, loading, error, refetch } = OrganizationEmployee.useObject(
         {
             where: {
                 id: employeeId,
             },
         }
     )
+
     const { objs: organizationEmployeeSpecializations } = OrganizationEmployeeSpecialization.useObjects({
         where: {
             employee: { id: employeeId },
         },
     })
 
+    const { data: activeTicketsOrganizationEmployeeCount } = useGetOrganizationEmployeeTicketsCountForReassignQuery({
+        variables: {
+            userId: employee?.user?.id,
+            organizationId: employee?.organization?.id,
+        },
+    })
+
     const employeeWithSpecializations = { ...employee, specializations: organizationEmployeeSpecializations.map(scope => scope.specialization) }
 
     const updateEmployeeAction = OrganizationEmployee.useUpdate({}, () => refetch())
-    const softDeleteAction = OrganizationEmployee.useSoftDelete(() => Router.push('/employee/'))
+    const softDeleteAction = OrganizationEmployee.useSoftDelete(() => refetch())
 
     const isEmployeeEditable = get(link, ['role', 'canManageEmployees'], false)
     const isEmployeeReinvitable = get(link, ['role', 'canInviteNewOrganizationEmployees'], false) && !get(employee, 'isAccepted')
 
-    if (error || loading) {
-        return <LoadingOrErrorPage title={UpdateEmployeeMessage} loading={loading} error={error ? ErrorMessage : null}/>
+    if (error || loading || (!loading && !employee)) {
+        const errorToPrint = error ? ErrorMessage : (!loading && !employee) ? NotFoundMsg : ''
+        return <LoadingOrErrorPage title={loading ? LoadingInProgressMessage : errorToPrint} loading={loading} error={errorToPrint}/>
     }
```

Comment:
```suggestion
    if (error || loading) {
        const errorToPrint = error ? ErrorMessage : ''
        return <LoadingOrErrorPage title={loading ? LoadingInProgressMessage : errorToPrint} loading={loading} error={errorToPrint}/>
    }
    
    if (!employee) {
        return <LoadingOrErrorPage title={errorToPrint} loading={false} error={NotFoundMsg}/>
    }
```

It's a little easier to read this way